hawkular-integrated-inventory-rest/src/main/resources/hawkular-inventory.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2015 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+storage.backend=cassandra
```

Comment:
Please a) synchronize that with other sub-projects.
See e.g. http://www.hawkular.org/docs/components/metrics/configuration.html
and
allow to set hostname + port via envronment variables (as indicated in above url).
Also HAWKULAR_BACKEND env variable is used in embedded cassandra to decide if this runs or not. 
Not sure if the above is related to it or rather switches between sql/C* backends.